modules/gdscript/gdscript_parser.cpp

```diff
@@ -3842,6 +3843,11 @@ bool GDScriptParser::validate_annotation_arguments(AnnotationNode *p_annotation)
 	return true;
 }
 
+bool GDScriptParser::uid_annotation(const AnnotationNode *p_annotation, Node *p_target, ClassNode *p_class) {
+	// Stub.
+	return true;
```

Comment:
What would we warn about?

Reply:
I'll copy what @YuriSizov [said to add in the documentation](https://github.com/godotengine/godot/pull/87475#discussion_r1462192468).

Reply:
Unsure specifically, but the file is from a higher version, granted a project won't necessarily be happy if opened in an older version so side-effects might be expected anyway

Reply:
Ah, you mean like a comment with an explanation? Yeah, that would be a good idea.

Reply:
> Unsure specifically, but the file is from a higher version, granted a project won't necessarily be happy if opened in an older version so side-effects might be expected anyway

If you mean a user-facing warning, then it would defeat the purpose of adding the annotation stub.

Reply:
A script editor warning perhaps but a comment too that it's "intentionally left blank" might be good too

Reply:
@AThousandShips This feature is for plugins that could be cross-compatible with 4.1, 4.2 and 4.3.

Reply:
Oh yes if it's for plugins it makes sense to be quiet and just ignore 🙂